### DISCUSSION TITLE
Regarding the nilfs2 file system not appearing in the disk list?

### DISCUSSION THREAD

**Discussion by llc0930** [ClementTsang/bottom#1595] [Q&A] [answered]
![2024-09-12-17:36:46-r](https://github.com/user-attachments/assets/6c492cca-8576-4078-889e-bd4de1fa7f70)
![2024-09-12-17:37:03-r](https://github.com/user-attachments/assets/cc7bae00-a708-4b21-ae06-afa136fc55ed)

Is it because of this?
https://github.com/ClementTsang/bottom/blob/eaa56238be0d2a6fc980175e7c0079dcb1d4ae65/src/data_collection/disks/unix/file_systems.rs#L160
What if it is changed to this?
`} else if multi_eq_ignore_ascii_case!(s, "nilfs" | "nilfs2") {`

**Comment by ClementTsang** [comment, marked as answer]
I suppose so. I don't use this file structure so very possible I missed this.

**Reply by llc0930** [reply]
Yes, it works now.

![2024-09-13-04:11:12-r](https://github.com/user-attachments/assets/9053c771-ac3d-487b-ae91-2d8881e448c2)
